app.go

```diff
@@ -685,10 +677,10 @@ func setGenesisDefaults() {
 	inflation.DefaultInflationState = mintDefaultInflationState()
```

Comment:
About the todo question of whether this is needed: By all means, if the SDK calls default genesis state initializers you may remove the explicit calls. I presume the module calls occur through the AppModule interface?